README.md

```diff
@@ -1,13 +1,116 @@
 # 💪 strong-config
 
+<!-- markdownlint-disable line-length -->
+[![pipeline status](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/pipeline.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master) [![coverage report](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/coverage.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+<!-- markdownlint-enable line-length -->
+
 Secret management for NodeJS. Based on [SOPS](https://github.com/mozilla/sops).
 
-[![pipeline status](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/pipeline.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+Have you ever pushed secrets to your git remote by accidentially committing
+the `.env` file? Then `strong-config` (built by Brickblock) is here to help:
```

Comment:
Good start, I would add some more examples here maybe to make this point stronger.

You should find some content in my [blog post drafts](https://www.notion.so/brickblock/Part-1-env-files-suck-3f5e70e599a04f889790fc49dd831caa)

Reply:
Ouh first time hearing about those drafts. Thx